### ISSUE TITLE
App crashes when looking for an artist

### ISSUE THREAD

**Issue by Miguel-Herrero** [metabrainz/musicbrainz-android#46]
Steps to reproduce:
1. Clean build of Master branch.
2. Run in Pixel 2 API 29
3. Write “Nathan Milstein” in the Artist search input.
4. Click on Search icon. Select first result (violinist)
5. Either app goes back to main screen, or it crashes.

Expected: go to Artist info.

## More Info: 
- Console log: [artist_lookup_nathan_milstein.log](https://github.com/metabrainz/musicbrainz-android/files/4844548/artist_lookup_nathan_milstein.log)
- File view with some errors in finding some views: <img width="300" alt="Captura de pantalla 2020-06-29 a las 10 10 50" src="https://user-images.githubusercontent.com/465723/85989463-d1b7e180-b9f0-11ea-881e-ac4ebc91c8d1.png">

- Android Studio info:

> Android Studio 3.6.3
> Build #AI-192.7142.36.36.6392135, built on April 14, 2020
> Runtime version: 1.8.0_212-release-1586-b4-5784211 x86_64
> VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
> macOS 10.15.5
> GC: ParNew, ConcurrentMarkSweep
> Memory: 1981M
> Cores: 8
> Registry: ide.new.welcome.screen.force=true
> Non-Bundled Plugins: org.jetbrains.kotlin, com.google.services.firebase

**Comment by amCap1712** [comment]
The issue was caused due to a bug in data binding generation which only occurs when built with Gradle older than 4.0. The error occurs when an include layout has an ID defined both in the original layout and where it has been included. Fixed in commit #39a6fd5 .

**Comment by Miguel-Herrero** [comment]
Now I can open an Artist. However when I click on one of its releases the app crashed, too. Could you check if this error is the same or related? 

This is the log of opening the first of “Justin Bieber”'s release:

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.metabrainz.android, PID: 15226
    java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.
        at android.view.ViewGroup.addViewInner(ViewGroup.java:5106)
        at android.view.ViewGroup.addView(ViewGroup.java:4935)
        at androidx.viewpager.widget.ViewPager.addView(ViewPager.java:1485)
        at android.view.ViewGroup.addView(ViewGroup.java:4875)
        at android.view.ViewGroup.addView(ViewGroup.java:4848)
        at org.metabrainz.mobile.presentation.features.release.CoverArtSlideshowAdapter.instantiateItem(CoverArtSlideshowAdapter.java:35)
        at androidx.viewpager.widget.ViewPager.addNewItem(ViewPager.java:1010)
        at androidx.viewpager.widget.ViewPager.populate(ViewPager.java:1158)
        at androidx.viewpager.widget.ViewPager.setCurrentItemInternal(ViewPager.java:669)
        at androidx.viewpager.widget.ViewPager.setCurrentItemInternal(ViewPager.java:631)
        at androidx.viewpager.widget.ViewPager.dataSetChanged(ViewPager.java:1086)
        at androidx.viewpager.widget.ViewPager$PagerObserver.onChanged(ViewPager.java:3097)
        at androidx.viewpager.widget.PagerAdapter.notifyDataSetChanged(PagerAdapter.java:291)
        at org.metabrainz.mobile.presentation.features.release.ReleaseInfoFragment.setCoverArt(ReleaseInfoFragment.java:75)
        at org.metabrainz.mobile.presentation.features.release.ReleaseInfoFragment.lambda$WuzlAqdDX2Fl6tdTPuSwMfOkieE(Unknown Source:0)
        at org.metabrainz.mobile.presentation.features.release.-$$Lambda$ReleaseInfoFragment$WuzlAqdDX2Fl6tdTPuSwMfOkieE.onChanged(Unknown Source:4)
        at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
        at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:149)
        at androidx.lifecycle.LiveData.setValue(LiveData.java:307)
        at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
        at androidx.lifecycle.Transformations$2$1.onChanged(Transformations.java:155)
        at androidx.lifecycle.MediatorLiveData$Source.onChanged(MediatorLiveData.java:152)
        at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
        at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:149)
        at androidx.lifecycle.LiveData.setValue(LiveData.java:307)
        at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
        at org.metabrainz.mobile.data.repository.LookupRepository$4.onResponse(LookupRepository.java:122)
        at retrofit2.DefaultCallAdapterFactory$ExecutorCallbackCall$1.lambda$onResponse$0$DefaultCallAdapterFactory$ExecutorCallbackCall$1(DefaultCallAdapterFactory.java:81)
        at retrofit2.-$$Lambda$DefaultCallAdapterFactory$ExecutorCallbackCall$1$3wC8FyV4pyjrzrYL5U0mlYiviZw.run(Unknown Source:6)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by amCap1712** [comment]
Another bug in view binding due to passing binding instead of the root view. Fixed in 28e56471ff31bb63ff03c4db6db11549e09ba6c1.
